File: my-submissions/m3064 v1.py
# Definition of commonSetBits API.
# def commonSetBits(num: int) -> int:

class Solution:
    def findNumber(self) -> int:
        output = 0
        current = 1
        for i in range(31) :
            if commonSetBits(current) :
                output += current
            current *= 2
        return output

File: my-submissions/m3064 v1.c
/** 
 * Definition of commonSetBits API.
 * int commonSetBits(int num);
 */

int findNumber(){
	int x = 1; // I'm assuming that this iteration of C is 32-bit ints
    int output = 0;

    for (int i = 0; i < 30; i++) {
        if (commonSetBits(x)) {
            output += x;
        }
        x = x << 2;
    }
    return output;
}